tests/test_l2_basic.py

```diff
@@ -383,6 +383,192 @@ def test_l2_lag(npu, dataplane):
             npu.set(oid, ["SAI_PORT_ATTR_PORT_VLAN_ID", npu.default_vlan_id])
 
 
+def test_l2_lag_hash_seed(npu, dataplane):
+    """
+    Description:
+    Check that the packets are equally divided among LAG members.
+
+    Test scenario:
+    1. Create a LAG group with 4 ports 1 through 4.
+    2. Setup static FDB entries for the LAG and send packet to this destination MAC address.
+    3. Send 100 packets with varying 5-tuple and check order/sequence of the distribution of packets received on ports 1 through 4.
+    4. Change the LAG Hash seed value to 10 and compare the order/sequence of the distribution of packets received for the same set of 100 packets on ports 1 through 4.
+    5. Verify that it is different after changing the hash seed.
+    """
+    vlan_id = "10"
+    mac = '00:11:11:11:11:11'
+    max_port = 4
+    lag_hashseed_value = "10"
+    lag_mbr_oids = []
+
+    # Remove bridge ports
+    for idx in range(max_port):
+        npu.remove_vlan_member(npu.default_vlan_oid, npu.dot1q_bp_oids[idx])
+        npu.remove(npu.dot1q_bp_oids[idx])
+
+    # Remove Port #4 from the default VLAN
+    npu.remove_vlan_member(npu.default_vlan_oid, npu.dot1q_bp_oids[4])
+
+    # Create LAG
+    lag_oid = npu.create(SaiObjType.LAG, [])
```

Comment:
Hm.. should we make `def create(..., attrs=[], .. )`? what do you thing?

Reply:
Added a "draft" version of create_lag API, please review

Reply:
Added a "draft" version of create_lag API with attrs=[] by default which simplifies the LAG creation and management. Please review